media-video/jubler/jubler-7.0.3-r1.ebuild

```diff
@@ -22,7 +22,7 @@ KEYWORDS="~amd64 ~x86"
 CP_DEPEND="dev-java/appenh:0"
 
 DEPEND="${CP_DEPEND}
-	media-video/ffmpeg:0=
+	media-video/ffmpeg:0/56.58.58
```

Comment:
Please do <media-video/ffmpeg-... instead as it's easier to read in portage output

Reply:
did you mead putting `<media-video/ffmpeg-5.1.3:0/56.58.58`? Or did you mean to drop the [,sub]slot dependency?

Reply:
Drop the subslot bit, yeah